Assets/MXR.SDK/Runtime/Android/Utils/MXRAndroidUtils.Apps.cs

```diff
@@ -92,16 +97,67 @@ public static void LaunchAppWithPackageAndClassNames(string packageName, string
         public static void LaunchIntentAction(string intentAction) =>
             NativeUtils.SafeCall<bool>("launchIntentAction", intentAction);
 
+        /// <summary>
+        /// The package name of the Admin App installed on an Android device
+        /// </summary>
+        /// <returns>Returns null if unsuccessful</returns>
+        public static string GetAdminAppPackageName() {
+            if(NativeUtils != null)
+                return NativeUtils.SafeCall<string>("getInstalledAdminAppPackageName");
+            return null;
+        }
 
-        public static string GetAdminAppPackageName() =>
-            NativeUtils.SafeCall<string>("getInstalledAdminAppPackageName");
-
+        /// <summary>
+        /// The version code of the Admin App installed on an Android device
+        /// </summary>
+        /// <returns>Returns -1 if unsuccessful</returns>
         public static int GetAdminAppVersionCode() {
             if (NativeUtils != null)
                 return NativeUtils.SafeCall<int>("getInstalledAdminAppVersionCode");
             return -1;
         }
 
+        /// <summary>
+        /// The version name string of the Admin App installed on an Android device
+        /// </summary>
+        /// <returns>Returns null if unsuccessful</returns>
+        public static string GetAdminAppVersionName() {
+            if (NativeUtils != null)
+                return NativeUtils.SafeCall<string>("getInstalledAdminAppVersionName");
+            return null;
+        }
+
+        /// <summary>
+        /// The <see cref="Version"/> of the Admin App installed on an Android device
+        /// </summary>
+        /// <returns>Returns null if the version name of the installed Admin App could not be retrieved
+        /// or if the retrieved value is invalid.
+        /// </returns>
+        public static Version GetAdminAppVersion() {
+            var versionName = GetAdminAppVersionName();
+            if (versionName == null)
+                return null;
+
+            // The version name may have a hyphen, e.g. "1.0.0-test"
+            var versionString = versionName.Split('-')[0];
+            if (Version.TryParse(versionString, out var version)) 
+                return version;
+            return null;
+        }
```

Comment:
There are probably libraries you could rely on to coerce something into a Semver version, but up to you if you think this approach is equally effective without the bloat!

Reply:
Did think about it, especially if a library could get the "test" from "1.0.0-test" but decided against it for bloat. The `Version` class is pretty easy to use and if we need more we can add a lib or consider making a utility